src/Alpha_complex/test/Incremental_delaunay_test.cpp

```diff
@@ -0,0 +1,87 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Marc Glisse
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE "incremental_delaunay"
+#include <boost/test/unit_test.hpp>
+#include <boost/range/adaptor/sliced.hpp>
+
+#include <CGAL/Epick_d.h>
+//#include <CGAL/Epeck_d.h>
+
+#include <vector>
+
+#include <gudhi/Incremental_delaunay.h>
+#include <gudhi/Simplex_tree.h>
+#include <gudhi/random_point_generators.h>
+#include <gudhi/Alpha_complex.h>
+
+using Static_kernel_3 = CGAL::Epick_d< CGAL::Dimension_tag<3> >;
+using Dynamic_kernel = CGAL::Epick_d< CGAL::Dynamic_dimension_tag >;
+using list_of_kernel_3_variants = boost::mpl::list<Static_kernel_3, Dynamic_kernel>;
```

Comment:
Should we also test `Epeck_d` here ? Or there is no added value ?

Reply:
There shouldn't be any added value to using Epeck_d, but that doesn't mean there cannot be in practice, and it wouldn't hurt to test at least one Epeck_d variant.